docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,432 @@
+              volumeSize: 120
+              volumeType: gp2
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: cuppett-tkh7l-worker-profile
+          instanceType: m4.large
+          kind: AWSMachineProviderConfig
+          metadata:
+            creationTimestamp: null
+          placement:
+            availabilityZone: us-east-2a
+            region: us-east-2
+          publicIp: null
+          securityGroups:
+          - filters:
+            - name: tag:Name
+              values:
+              - cuppett-tkh7l-worker-sg
+          subnet:
+            filters:
+            - name: tag:Name
+              values:
+              - cuppett-tkh7l-private-us-east-2a
+          tags:
+          - name: kubernetes.io/cluster/cuppett-tkh7l
+            value: owned
+          userDataSecret:
+            name: worker-user-data
+      versions:
+        kubelet: ""
+status:
+  fullyLabeledReplicas: 1
+  observedGeneration: 1
+  replicas: 1
+```
+
+At this point, you'd edit the YAML to update the relevant values to match your UPI installation.
+
+```console
+$ oc edit machineset --namespace openshift-machine-api cuppett-tkh7l-worker-us-east-2a
+machineset.machine.openshift.io/cuppett-tkh7l-worker-us-east-2a edited
+```
+
+Once the Machine API has a chance to reconcile and begin launching hosts with the correct attributes, you 
+should start to see new output in your EC2 console and oc commands.
+
+```console
+$ oc get machines --namespace openshift-machine-api
+NAME                                    INSTANCE              STATE     TYPE         REGION      ZONE         AGE
+cuppett-tkh7l-worker-us-east-2a-hxlqn   i-0e7f3a52b2919471e   pending   m4.4xlarge   us-east-2   us-east-2a   3s
+```
+
+### Option 2: Manually Launching Worker Instances
+
+The recommended worker launch and other necessary DNS entries for etcd are provided within a CloudFormation
+template [here](../../../upi/aws/cloudformation/06_cluster_worker_node.yaml). You can launch a CloudFormation stack
+to manage each individual worker. A similar launch configuration could be used by outside automation or AWS auto
+scaling groups. 
+
+## Configure Router for UPI DNS
+
+TODO: Identify changes needed to Router or Ingress for DNS *.apps registration or LoadBalancer creation.
+
+## Monitor for cluster completion
+
+```console
+$ bin/openshift-install user-provided-infrastructure finish
+INFO Waiting up to 30m0s for the cluster to initialize...
+```
+
+Also, you can observe the running state of your cluster pods:
+
+```console
+$ oc get pods --all-namespaces
+NAMESPACE                                               NAME                                                                READY     STATUS      RESTARTS   AGE
+kube-system                                             etcd-member-ip-10-0-3-111.us-east-2.compute.internal                1/1       Running     0          35m
```

Comment:
This output is pretty noisy.  I'd rather drop "Also, you can observe..." and this whole block, but if you want to keep it, maybe just keep a few lines of pods and then drop an ellipsis?  I think a better approach would be to have `waitForInitializedCluster` be a bit more noisy about logging occasional status updates (currently it only has debug-level logging for failures).

Reply:
Fixed.